roles/python_interpreter/tasks/main.yml

```diff
@@ -1,11 +1,11 @@
 - block:
   - name: Get Ubuntu release
-    raw: lsb_release -cs
-    register: ubuntu_release
+    raw: lsb_release -rs
+    register: ubuntu_version
     changed_when: false
-  - name: Set ansible_python_interpreter for Ubuntu 18.04 Bionic
+  - name: Set ansible_python_interpreter for Ubuntu >= 18.04
     set_fact:
       ansible_python_interpreter: python3
-    when: ubuntu_release.stdout | trim == 'bionic'
+    when: ubuntu_version.stdout | trim is version('18.04', '>=')
```

Comment:
How about we drop support for 18.04?
Then, we don't need to worry about python 2

Reply:
🤔 it would simplify this, but we still might want to support python 2 locally? So most of the work to make Trellis python compatible would remain if we want to keep that local support.